src/test/ui/rfc-2093-infer-outlives/self-structs.stderr

```diff
@@ -6,7 +6,7 @@ LL | |     field1: dyn Bar<'a, 'b, T>
 LL | | }
    | |_^
    |
-   = note: T : 'a
+   = note: T: 'a
```

Comment:
Can you move the change that causes all of these test output changes into another PR? LGTM but I'd want to keep this PR minimal.

Reply:
Did that in https://github.com/rust-lang/rust/pull/68517